Engine/Component/Component.h

```diff
@@ -26,11 +26,13 @@ class Component
 		CANVAS_RENDERER = 10,
 		UI_BUTTON = 11,
 		UI_IMAGE = 12,
-		UI_SPRITE_MASK = 18,
+		UI_SPRITE_MASK = 20,
```

Comment:
this change is yours or is from the merge? we have to be very careful with these numbers

Reply:
I think its from the merge. I added only TRAIL&TRAILRENDERER.